acoustid/future/fpindex/updater.py

```
@@ -0,0 +1,172 @@
+                delay = min(delay * 2, REPL_MAX_DELAY)
+
+            # if we processed less than the full batch size, wait a bit
+            if len(changes) < REPL_BATCH_SIZE:
+                logger.info("Waiting %s seconds", delay)
+                await asyncio.sleep(delay)
+
+
+@click.group()
+def main() -> None:
+    logging.basicConfig(level=logging.INFO)
+    pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)